src/core/js/a11y/popup.js

```diff
@@ -0,0 +1,212 @@
+            $focusElement = $focusElement || $previousFocusElement || $('body');
+            if (!silent) {
+                Adapt.trigger('popup:closed', $focusElement, true);
+            }
+            Adapt.a11y.focusFirst($($focusElement));
+            return this;
+        },
+
+        /**
+         * Restores tabbing and screen reader access to the state before the last
+         * `_addPopupLayer` call.
+         *
+         * @returns {Object} Returns previously active element.
+         */
+        _removeLastPopupLayer: function() {
+            var config = Adapt.a11y.config;
+            if (!config._isEnabled || !config._isPopupManagementEnabled) {
+                return $(document.activeElement);
+            }
+            // the body layer is the first element and must always exist
+            if (this._floorStack.length <= 1) {
+                return;
+            }
+            this._floorStack.pop();
+            $(config._tabbableElements).filter(config._tabbableElementsExcludes).each(function(index, item) {
+                var $item = $(item);
+                var previousTabIndex = '';
+                var previousAriaHidden = '';
+                var elementUID;
+                if (item.a11y_uid == undefined) {
+                    //assign element a unique id
+                    item.a11y_uid = 'UID' + ++this._elementUIDIndex;
+                }
+                elementUID = item.a11y_uid;
+                if (this._tabIndexes[elementUID] !== undefined && this._tabIndexes[elementUID].length !== 0) {
+                    //get previous tabindex if saved
+                    previousTabIndex = this._tabIndexes[elementUID].pop();
+                    previousAriaHidden = this._ariaHiddens[elementUID].pop();
+                }
+                if (this._tabIndexes[elementUID] !== undefined && this._tabIndexes[elementUID].length === 0) {
+                    //delete element tabindex store if empty
+                    delete this._tabIndexes[elementUID];
+                    delete this._ariaHiddens[elementUID];
+                }
+                if (config._isPopupTabIndexManagementEnabled) {
+                    if (previousTabIndex === '') {
+                        $item.removeAttr('tabindex');
+                    } else {
+                        $item.attr({
+                            'tabindex': previousTabIndex
+                        });
+                    }
+                }
+                if (config._isPopupAriaHiddenManagementEnabled) {
+                    if (previousAriaHidden === '') {
+                        $item.removeAttr('aria-hidden');
+                    } else {
+                        $item.attr({
+                            'aria-hidden': previousAriaHidden
+                        });
+                    }
+                }
+            }.bind(this));
+            return document.activeElement = this._focusStack.pop();
```

Comment:
wrap in brackets to stop linter from complaining
```suggestion
            return (document.activeElement = this._focusStack.pop());
```